google/cloud/bigtable/examples-stock-backtesting/WORKSPACE.bazel

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A minimal WORKSPACE file showing how to use the Google Cloud Bigtable C++
+# client library in Bazel-based projects.
+workspace(name = "bigtable_example_stock_backtesting")
+
+# Add the necessary Starlark functions to fetch google-cloud-cpp.
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# Fetch the Google Cloud C++ libraries.
+# NOTE: Update this version and SHA256 as needed.
+http_archive(
+    name = "com_github_googleapis_google_cloud_cpp",
+    sha256 = "f38ae4ab6f2ed7579a7ceb5d0b32ed04097da07bc898907ed01c8d840c2bdbce",
+    strip_prefix = "google-cloud-cpp-1.34.1",
+    url = "https://github.com/googleapis/google-cloud-cpp/archive/v1.34.1.tar.gz",
+)
+
+# Fetch the ABSL C++ libraries.
```

Comment:
since Abseil is an direct dependency of `google-cloud-cpp` you can save yourself this definition.